pkg/csi/service/fcd/controller.go

```diff
@@ -84,11 +85,11 @@ func (c *controller) CreateVolume(
 
 	//check for required parameters
 	if params == nil {
-		msg := fmt.Sprintf("Create parameters is a required parameter.")
+		msg := "Create parameters is a required parameter."
 		log.Errorf(msg)
```

Comment:
can also change all the `Errorf` to `Error`

Reply:
Will fix